pkg/openapiclient/local_files.go

```diff
@@ -123,7 +123,7 @@ func (k *localFilesClient) Paths() (map[string]openapi.GroupVersion, error) {
 
 		for _, v := range versions {
 			ext := strings.ToLower(filepath.Ext(v.Name()))
-			if ext != "json" {
+			if ext != ".json" {
```

Comment:
Similarly this can use

https://github.com/kubernetes-sigs/kubectl-validate/blob/27b2bf459cd8c677b4244cd954f87d66fc5102d8/pkg/utils/files.go#L14-L17